src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -195,15 +196,15 @@ export default class AircraftController {
             // let is_visible = aircraft_visible(aircraft);
 
             if (aircraft.isStopped() && aircraft.category === 'arrival') {
-                window.gameController.game.score.windy_landing += aircraft.scoreWind('landed');
+                aircraft.scoreWind('landed');
```

Comment:
This is beyond the scope of your changes, but why is the `scoreWind` method being used on an `aircraft` instance?  Looking at the method:
```javascript
    scoreWind(action) {
        let score = 0;
        const components = this.getWind();
        const isWarning = true;

        // TODO: these two if blocks could be done in a single switch statement
        if (components.cross >= 20) {
            window.gameController.game_add_to_score(GAME_EVENTS.EXTREME_CROSSWIND_OPERATION);
            window.uiController.ui_log(`${this.getCallsign()} ${action} with major crosswind'`, isWarning);
        } else if (components.cross >= 10) {
            window.gameController.game_add_to_score(GAME_EVENTS.HIGH_CROSSWIND_OPERATION);
            window.uiController.ui_log(`${this.getCallsign()} ${action} with crosswind'`, isWarning);
        }

        if (components.head <= -10) {
            window.gameController.game_add_to_score(GAME_EVENTS.EXTREME_TAILWIND_OPERATION);
            window.uiController.ui_log(`${this.getCallsign()} ${action} with major tailwind'`, isWarning);
        } else if (components.head <= -1) {
            window.gameController.game_add_to_score(GAME_EVENTS.HIGH_TAILWIND_OPERATION);
            window.uiController.ui_log(`${this.getCallsign()} ${action} with tailwind'`, isWarning);
        }

        return score;
    }
```

I don't see much in that method that the `AircraftInstanceModel` should be concerned about.  

Could you add a TODO above the `scoreWind` method that it needs to be moved/refactored?